--- parse.y-	2014-09-14 15:41:23.201932612 +0900
+++ parse.y	2014-09-14 15:41:48.281844882 +0900
@@ -564,7 +564,7 @@
 		| kTHEN | kTRUE | kUNDEF | kUNLESS_MOD | kUNTIL_MOD | kWHEN
 		| kWHILE_MOD | kYIELD
 
-arg		: variable '=' {$$ = assignable($1, 0);} arg
+arg		: variable '=' {$<node>$ = assignable($1, 0);} arg
 		    {
 			$$ = $<node>3;
 			if ($$) {
@@ -593,7 +593,7 @@
 		        rb_backref_error($1);
 			$$ = 0;
 		    }
-		| variable tOP_ASGN {$$ = assignable($1, 0);} arg
+		| variable tOP_ASGN {$<node>$ = assignable($1, 0);} arg
 		    {
 			if ($2 == tOROP) {
 			    $<node>3->nd_value = $4;
